docs/links.md

```diff
@@ -40,8 +40,9 @@
 
 ### Tools
 
-- [Forte](https://github.com/MyBlackMIDIScore/forte) - ⭐ Convert MIDI files to MP3, WAV, or OGG using Xsynth.
-- [Keppy's MIDI Converter](https://github.com/KeppySoftware/KMC) - ⭐ Convert MIDI files to MP3, WAV, or OGG.
+- [Forte](https://github.com/MyBlackMIDIScore/forte) - ⭐ Convert MIDI files to MP3, WAV, or OGG using Xsynth. (Requires compilation)
+- [Keppy's MIDI Converter](https://github.com/KeppySoftware/KMC) - Convert MIDI files to MP3, WAV, or OGG. (Replaced by OmniConverter)
+- [OmniConverter](https://github.com/KaleidonKep99/OmniConverter) - ⭐ Convert MIDI files to MP3, WAV, or OGG.
```

Comment:
I still don't like the star on OmniConverter. IMO since you need compilation might as well go with Forte instead, no?

```suggestion
- [Keppy's MIDI Converter](https://github.com/KeppySoftware/KMC) - ⭐ Convert MIDI files to MP3, WAV, or OGG. (Replaced by OmniConverter)
- [OmniConverter](https://github.com/KaleidonKep99/OmniConverter) - Convert MIDI files to MP3, WAV, or OGG.
```